### ISSUE TITLE
noto.fyi down? Notes are not sending from Noto

### ISSUE THREAD

**Issue by ianianianianian** [notofyi/org#533]
Noto doesn't seem to be sending notes. I have about 4 in queue that just hang on Sending...

Also Noto.fyi seems to be down.

Is Noto no longer being supported? 

Thanks

https://imgur.com/a/uJcrZF8

**Comment by danielsuo** [comment]
Yep, noto has been turned down. There is someone else looking to build a
new version.

**Comment by ianianianianian** [comment]
Damn. Thanks for the cool app though. :)
